nidm_results_display.m

```diff
@@ -14,7 +14,10 @@
 
     %Record users choice and filepath.
     jsondoc=spm_jsonread(jsonfilepath);
-    [pathstr, str] = fileparts(jsonfilepath); 
+    [pathstr, str] = fileparts(jsonfilepath);
+    
+    %We expect the files for the nidm pack to be stored in a folder, of the
+    %same name, located next to the jsons folder.
```

Comment:
May I suggest we leave this until a later update, as all NIDM packs which store a json need the updates from PR20 to run. Therefore, I cannot test whether this would work correctly until PR20 is merged.